### PR TITLE
[doc] Updated document on max map count

### PR DESCRIPTION
**Pull request by writinwaters**
### What problem does this PR solve?

_Briefly describe what this PR aims to solve. Include background context that will help reviewers understand the purpose of the PR._

### Type of change

- [x] Documentation Update

### CODE REVIEW

File: docs/guides/max_map_count.md [hunk 1]
```diff
@@ -3,69 +3,92 @@ sidebar_position: 7
 slug: /max_map_count
 ---
 
-# Update vm.max_map_count
+# Update maximum memory map areas
```
Comment (KevinHuSh) [inline docs/guides/max_map_count.md]: # Update vm.max_map_count

**Review by KevinHuSh — APPROVED** [review]

**Comment by writinwaters** [comment]
Merged max_map_count.md into quickstart:
- Expand view: 
expand

- Collapse view
collapse

File: docs/quickstart.mdx [hunk 2]
```diff
@@ -4,6 +4,8 @@ slug: /
 ---
 
 # Quick start
+import Tabs from '@theme/Tabs';
```
Comment (JinHai-CN) [inline docs/quickstart.mdx:7]: What is this?

Comment (JinHai-CN) [inline docs/quickstart.mdx:7]: What is this?

**Review by JinHai-CN — APPROVED** [review]

**Pull request merged**